ui/src/state/mod.rs

```diff
@@ -96,6 +101,104 @@ impl State {
         State::default()
     }
 
+    pub fn mock() -> Self {
```

Comment:
since commit `11b88c3050e9278c39de38871df27168de6a14a7`, `State::new()` and `State::mock` are unnecessary. all that was handled by the `load()` function. it got broken during a merge with dev. using the `load()` function for everything prevents bugs like the one mentioned in `main.rs`